webpack.config.dev.js

```diff
@@ -34,7 +36,14 @@ module.exports = {
     auth: ['webpack-hot-middleware/client', path.resolve(__dirname, srcScript, 'auth/main.tsx')],
   },
   mode: 'development',
-  plugins: [...commonConfig.plugins, new webpack.HotModuleReplacementPlugin()],
+  plugins: [
+    ...commonConfig.plugins,
+    new webpack.HotModuleReplacementPlugin(),
+    new DeadCodePlugin({
+      patterns: ['src/**/*.(js|jsx|css|ts|tsx)'],
+      exclude: ['**/*.(stories|spec|test).(js|jsx|ts|tsx)', 'src/types/*.*', '**/*.d.ts', '**/__mocks__'],
+    }),
```

Comment:
I don't think we need to add it at this point. This might slow down dev workflow a bit and we almost never look at the webpack report anyway. 
It's okay if files are not consumed (and actually there are a lot of false positive in the report it generated, like mock for testing)

Reply:
Should I just create a Jira for removing the unused code using this report (excluding mocks)?